src/Messenger/IndexationRequestSpoolSubscriber.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace JoliCode\Elastically\Messenger;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Transport\TransportInterface;
+
+/**
+ * Same idea as https://github.com/symfony/swiftmailer-bundle/blob/master/EventListener/EmailSenderListener.php.
+ */
+class IndexationRequestSpoolSubscriber implements EventSubscriberInterface
+{
+    private $logger;
+
+    private $wasExceptionThrown = false;
+
+    private $singleTransport;
+
+    private $bus;
+
+    public function __construct(TransportInterface $singleTransport, MessageBusInterface $bus, LoggerInterface $logger = null)
+    {
+        $this->logger = $logger;
+        $this->singleTransport = $singleTransport;
+        $this->bus = $bus;
+    }
+
+    public function onException()
+    {
+        $this->wasExceptionThrown = true;
+    }
+
+    public function onTerminate()
```

Comment:
Isn't expected to check `$this->wasExceptionThrown` here?

Reply:
Absolutely! Fixed, thanks!